apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.html.ts

```diff
@@ -204,32 +204,39 @@ export function InstructorAssessmentGroups({
                                     >
                                       <i class="fa fa-user-plus" aria-hidden="true"></i> Add members
                                     </button>
-                                    <button
-                                      class="dropdown-item js-group-action"
-                                      ${row.users.length === 0
-                                        ? 'disabled'
-                                        : html`
-                                            data-bs-toggle="popover" data-bs-container="body"
-                                            data-bs-html="true" data-bs-placement="auto"
-                                            title="Remove members"
+                                    ${row.users.length > 0
+                                      ? html`
+                                          <button
+                                            class="dropdown-item js-group-action"
+                                            data-bs-toggle="popover"
+                                            data-bs-container="body"
+                                            data-bs-html="true"
+                                            data-bs-placement="auto"
+                                            data-bs-title="Remove members"
```

Comment:
I changed the markup here to allow `html` to always contain a complete HTML element, which is better for syntax highlighting and formatting. Dynamic attributes like this is the kind of thing that JSX is much more suited for.